src/model/Deployment.ts

```diff
@@ -0,0 +1,29 @@
+import {IsNotEmpty, IsOptional} from 'class-validator';
+import {IsFridgeISO8601WithoutMs} from '../custom-validation-decorator/FridgeCustomValidationDecorators';
+
+export class Deployment {
+  @IsNotEmpty()
+  teamName: string;
+
+  @IsNotEmpty()
+  releaseTag: string;
+
+  @IsNotEmpty()
+  environment: string;
+
+  @IsNotEmpty()
+  applicationName: string;
+
+  @IsNotEmpty()
+  @IsFridgeISO8601WithoutMs()
+  timestamp: string;
```

Comment:
I was thinking that we'll add timestamp ourselves.

Reply:
This is the deployment timestamp and would prefer the user to insert it, because it's not the time they push to the api

Reply:
ok